var/spack/repos/builtin/packages/ftgl/package.py

```diff
@@ -16,10 +14,15 @@ class Ftgl(AutotoolsPackage):
     list_url = "https://sourceforge.net/projects/ftgl/files/FTGL%20Source/"
     list_depth = 1
 
-    version('2.1.2', 'f81c0a7128192ba11e036186f9a968f2')
+    version('2.1.3-rc5', sha256='5458d62122454869572d39f8aa85745fc05d5518001bcefa63bd6cbb8d26565b')
```

Comment:
Note (not a request): removing old versions is generally discouraged but ROOT is currently the only package using this.

Reply:
See https://github.com/spack/spack/pull/11214#discussion_r277428195